### PR TITLE
Licensing cleanup

### PR DESCRIPTION
**Pull request by keszybz**
Companion to #20886.

### CODE REVIEW

**Review by bluca — APPROVED** [review]
LGTM, added a comment in the README about BSD-3-Clause in https://github.com/systemd/systemd/pull/20886

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Bug report
 about: A report of an error in a recent systemd version
-
+SPDX-License-Identifier: LGPL-2.1-or-later
```
Comment (poettering) [inline .github/ISSUE_TEMPLATE/Bug_report.md:4]: hmm, this is weird. this is a template for to be created github issues. can we do this? add a license to everyone's issue submissions?

**Comment by poettering** [comment]
lgtm, but i wonder about the issue template

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 1 above)
Comment (bluca) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: This is metadata about the template itself, no? If you click on "new issue" you don't see these, they are not copied over.

Comment (bluca) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: ie: click here https://github.com/systemd/systemd/issues/new?assignees=&labels=&template=Bug_report.md and see that there's no "name" or "about", it's just metadata fields

**Pull request merged**